### ISSUE TITLE
Would you alos share your caffe training/val log file as well?

### ISSUE THREAD

**Issue by Coderx7** [DeepScale/SqueezeNet#39]
Hello, 
Thank you for your contribution and beautiful work. 
May I ask you to also share with us the log files for your training ? 
Thank you in advance

**Comment by Cospel** [comment]
This will be great, if you can share this.

I'm trying to train it from scratch on imagenet12 in tensorflow however my validation accuracy is lower(70% :( )
